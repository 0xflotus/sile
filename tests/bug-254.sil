\begin[papersize=a5]{document}
\raggedright{
\font[weight=600]{This} is a sample of bogus\break
extra space.

\font[weight=600]{This} does not have\break
{}extra space.
}
\end{document}