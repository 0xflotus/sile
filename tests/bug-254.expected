Set paper size 	419.5275636	595.275597
Begin page
Mx 	20.9764
My 	37.3321
Set font 	Gentium Plus;10;600;;normal;;LTR
T	55 75 76 86	(This)
Mx 	41.2166
Set font 	Gentium Plus;10;400;;normal;;LTR
T	76 86	(is)
Mx 	50.4315
T	68	(a)
Mx 	57.6639
T	86 68 80 83 79 72	(sample)
Mx 	89.4178
T	82 73	(of)
Mx 	100.2000
T	69 82 74 88 86	(bogus)
Mx 	20.9764
My 	49.3321
T	72 91 87 85 68	(extra)
Mx 	45.2596
T	86 83 68 70 72	(space)
Mx 	67.9988
T	17	(.)
Mx 	20.9764
My 	61.3321
Set font 	Gentium Plus;10;600;;normal;;LTR
T	55 75 76 86	(This)
Mx 	41.2166
Set font 	Gentium Plus;10;400;;normal;;LTR
T	71 82 72 86	(does)
Mx 	62.5701
T	81 82 87	(not)
Mx 	79.2020
T	75 68 89 72	(have)
Mx 	20.9764
My 	73.3321
T	72 91 87 85 68	(extra)
Mx 	45.2596
T	86 83 68 70 72	(space)
Mx 	67.9988
T	17	(.)
Mx 	207.4176
My 	553.7327
T	20	(1)
End page
Finish
